src/components/EntityInfo.vue

```diff
@@ -24,17 +24,6 @@
               <span>{{ entityName || getCorpTypeNumberedDescription(entityType)}}</span>
             </div>
 
-            <!-- Entity Status -->
-            <v-chip v-if="isGoodStanding" class="blue" id="entity-status" small label text-color="white">
```

Comment:
Can you please delete the now-obsolete getters?

Reply:
Done